lib/virtmanager.pm

```diff
@@ -15,9 +15,11 @@ sub launch_virtmanager() {
     type_string "virt-manager", 50;
     send_key "ret";
     wait_idle;
-    type_password;
-    send_key "ret";
-    wait_idle;
+    if (check_var("FLAVOR", "Server-DVD")) {
```

Comment:
I have ran a rough test on SLES12SP1to make sure these change won't destroy the Server's testing result. If needed, I will test it on SLES12 SP2 again